src/components/views/messages/EditHistoryMessage.tsx

```diff
@@ -141,13 +141,18 @@ export default class EditHistoryMessage extends React.PureComponent<IProps, ISta
             );
         }
 
-        // disabled remove button when not allowed
-        return (
-            <div className="mx_MessageActionBar">
-                {redactButton}
-                {viewSourceButton}
-            </div>
-        );
+        if (!redactButton && !viewSourceButton) {
+            // Hide the empty MessageActionBar
+            return null;
```

Comment:
The return type of `renderActionBar` should be updated to `ReactNode` to cover the null case.

Reply:
I see, please wait for a moment...

Reply:
Done.